lib/active_merchant/billing/gateways/cecabank/cecabank_json.rb

```diff
@@ -140,8 +140,23 @@ def add_stored_credentials(post, creditcard, options)
 
       def add_three_d_secure(post, options)
         params = post[:parametros] ||= {}
-
-        params[:ThreeDsResponse] = options[:three_d_secure] if options[:three_d_secure]
+        three_d_secure = options[:three_d_secure]
+        return unless three_d_secure.present?
+
+        three_d_response = {}
+        three_d_response[:exemption_type] = three_d_secure[:exemption_type]
+        three_d_response[:three_ds_version] = three_d_secure[:version]
+        three_d_response[:authentication_value] = three_d_secure[:cavv]
+        three_d_response[:directory_server_transaction_id] = three_d_secure[:ds_transaction_id]
+        three_d_response[:acs_transaction_id] = three_d_secure[:acs_transaction_id]
+        three_d_response[:authentication_response_status] = three_d_secure[:authentication_response_status]
+        three_d_response[:three_ds_server_trans_id] = three_d_secure[:three_ds_server_trans_id]
+        three_d_response[:ecommerce_indicator] = three_d_secure[:eci]
```

Comment:
💬 Comment:

You can add the `enrolled` attributed from `three_d_secure[:enrolled]`